docs/developers_guide/quick_start.rst

```diff
@@ -3,6 +3,18 @@
 Quick Start for Developers
 ==========================
 
+.. _dev_shell:
+
+Unix Shell
+----------
+
+Currently, compass only supports ``bash`` and related unix shells (such as
+``ksh`` on the Mac).  We do not support ``csh``, ``tcsh`` or other variants of
+``csh``.  An activation script for those shells will not be created.
+
+If you normally use ``csh``, ``tcsh`` or similar, you will need to temporarily
+switch to bash by calling ``/bin/bash`` each time you want to use compass.
+
```

Comment:
@milenaveneziani, feel free to have a look at this and make any suggestions.  I won't make you review it formally in case you don't have time.  I'll merge this in about half an hour so if you don't happen to have a chance to look at it, no worries.

Reply:
sorry, I was away for a few hours. But looks good to me. Thanks!